packages/fluent-ui/src/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -63,7 +72,7 @@ const CheckboxesWidget = ({
   }: React.FocusEvent<HTMLButtonElement>) => onFocus(id, value);
 
   return (
-    <>
+    <div>
```

Comment:
Can you remove the <div> and do <> instead?

Reply:
Done.